lib/data.js

```diff
@@ -23,39 +25,65 @@ const clone = (
   return Array.isArray(obj) ? cloneArray(obj) : cloneObject(obj);
 };
 
-function cloneArray(arr) {
-  const size = arr.length;
-  const array = new Array(size);
-  let i, val;
-  for (i = 0; i < size; i++) {
-    val = arr[i];
-    if (typeof(val) !== 'object' || val === null) {
-      array[i] = val;
-    } else if (Array.isArray(val)) {
-      array[i] = cloneArray(val);
+const transformArrObj = (
```

Comment:
Bad naming, it's not obvious what `transform` means. Whole name `transformArrObj` is not clear at all.

Reply:
Is `handle` suitable here? Could whole name be `handleArrOrObj`?